src/sdsc/__init__.py

```diff
@@ -184,7 +189,12 @@ def counttokens(context, text):
     :param str context: context node (ignored)
     :param str text: text in which to count tokens
     """
-    del context  # not used
+    del context  # not used # FIXME toms: better remove this line
```

Comment:
In that case, you could use `_` as variable name. Or, as another alternative, you could also disable this warning for those functions.